### ISSUE TITLE
Other Git lessons

### ISSUE THREAD

**Issue by ErinBecker** [carpentries-incubator/proposals#17]
- Mod of git-novice lessons - https://uw-madison-datascience.github.io/git-novice-custom/ - Teaches branches and pull requests

- Code refinery git intro https://coderefinery.github.io/git-intro/ - Guacamole example

- Git - Github - Rstudio https://inbo.github.io/git-course/

- Steve Bond (Biologyguy)'s Git and Github with examples that make more sense IRL: https://biologyguy.github.io/git-novice/
     - Two hour edition targeted at IT pros based on his version: https://github.com/dlstrong/git-novice/tree/it-pro-forum
 - I teach this as a full day lesson, and use a cut-down version for SWC workshops: http://gcapes.github.io/git-course
   - Writing a paper as the narrative, covers branches, revert, reset, rebase, remotes, PRs.

- The Open Science MOOC has a task as part of its module on Open Research Software and Open Source about Git and RStudio integration: https://eliademy.com/catalog/oer/module-5-open-research-software-and-open-source.html (available outside of the MOOC platform itself too if needed)

- Modified version of git-novice that uses Git Desktop and uses editing a carpentry data file as example: https://caltechlibrary.github.io/git-desktop/

**Comment by ErinBecker** [comment]
@uw-madison-datascience, @coderefinery, @inbo, @biologyguy, @dlstrong, @gcapes, @caltechlibrary - We've just created The Carpentries Incubator GH organization to provide a single point of entry for community members to share lesson materials and discuss ideas for new lessons. Your lesson materials were previously listed on the sharing-lessons Etherpad, which has now been deprecated. All of the resources listed on that pad have been migrated here. We're still working on organizing this repo, so please do let me know if you have ideas for how best to organize this information to make it accessible to the community.

**Comment by dlstrong** [comment]
Hi @ErinBecker ! I've got a 3 hour version that includes tagging, branches, and undoing things / getting other items out of history here: https://github.com/dlstrong/git-novice/tree/ITPF2019 

Segment 5 - rendered at https://dlstrong.github.io/git-novice/05-label-and-undo/ -  is what our IT pros asked for over several semesters of runs.

**Comment by ErinBecker** [comment]
Cool, thanks for sharing @dlstrong! If you'd like to have this lesson migrated to The Carpentries Incubator (and rendered at a carpentries-incubator url, please can you confirm the following:

- It uses The Carpentries lesson template
- It has an open license (CC-0 or CC-BY) and 
- It conforms to The Carpentries Code of Conduct

If all of the above are true, please let me know and I'll add you as a collaborator so you can transfer the repo. Let me know if you have any questions!

**Comment by dlstrong** [comment]
It uses what's probably an older version of the Carpentries lesson template - I haven't pulled all the updates from the source since I took my fork off in a different direction. Also I'd probably want to de-identify some of the examples in the text; references to lessons being taught in our campus computer labs aren't as helpful for the entire universe as they are for our campus people. Is there a way I can contribute a modified / universal-ized version for Carpentries and still keep my own fork campus-focused?

**Comment by ErinBecker** [comment]
Absolutely! Since GitHub only allows each organization/individual account to have a single fork of each repo, I think the best way to do this would be to use GitHub's importer to create a copy of the lesson in your account, and then to transfer one of those lessons to the Incubator. This should separate the history of those two repos and let you work on each of them independently. Please let me know if you'd like help bringing in the lesson template updates. There are some instructions here (http://carpentries.github.io/lesson-example/setup.html) for using GH's importer and updating from the template, but I'm also happy to help out - it's taken me a while to get the hang of this process! 😄

**Comment by ErinBecker** [comment]
Absolutely no rush! I'll make a note to myself to follow up with you in December. Hope things calm down for you a bit and please focus on the sleep!
